### PR TITLE
T-97/ Shipping Detail Unit Tests

### PR DESCRIPTION
**Pull request by fiorellamansilla**
- Implemented new GET methods in Shipping Detail Service 
- Created and implemented GET endpoints in Shipping Detail Controller
- Implemented Unit tests for all methods from Shipping Detail Service
- Updated dependencies on Build.gradle
- Created a README file

### CODE REVIEW

**Review by diego-mansilla — APPROVED** [review]
Cool, but it is better for a PR to only contain specific changes (Only the README changes and the code in a separate PR)

File: README.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+1. Clone the repository to your local machine:
+
+   ```shell
+   git clone https:https://github.com/fiorellamansilla/kuna_backend
+
+2. Navigate to the project directory:
+
+    ```shell
+   cd project-directory
```
Comment (diego-mansilla) [inline README.md:24]: This is not right

File: README.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+   
+   This configuration sets up a MySQL container with the necessary environment variables and exposes port 3306 <br />
+
+
+5. Start the MySQL container using Docker Compose:
+
+   ```shell
+   docker-compose up -d
+
+6. Create a MySQL database for the application.<br />
```
Comment (diego-mansilla) [inline README.md:60]: Ideally we should create scripts for this test data

File: README.md (hunk 1 above)
Comment (fiorellamansilla) [reply README.md:24]: Noted, I've made the changes.

**Pull request merged**